NEWS

```diff
@@ -11,6 +11,333 @@
 ### Features:
 ### Bugfixes:
 
+## 1.14.0 (January 12, 2023)
+### Features:
+#### Core
+#### UCP
+* Added API for querying of transport and device names on endpoint
+* Added datatype query API
+* Added new API ucp_memh_pack()
+* Added API to support exported memory handles (no implementation yet)
+* Added support for non persistent header in AM API
+* Added performance tree infrastructure for protocols v2
+* Added infrastructure to print protocols v2 performance
+* Multiple performance improvements for protocols v2
+* Added support for non-contiguous datatypes in AM rendezvous protocol
+* Added support for non-contiguous datatypes for rendezvous protocols v2
+* Added support for reset and abort request in protocols v2
+* Added rendezvous support for TAG sync send API in protocols v2
+* Added support for user memory handles in RMA API
+* Added multi-rail support for RMA API in protocols v2
+* Added support for up to 16 different lanes per endpoint
+* Added support for dmabuf memory registration in protocols v2
+* Added statistics to protocols v2
+* Added strong fence mode for ucp_worker_fence() API
+* Added counter for pending connection requests on listener
+#### UCT
+* Added new uct_md_mem_attach() API to support exported memory handles
+* Added remote completion mode for endpoint flush (via new flag)
+* Added support for dmabuf registration
+* Added new uct_ep_connect_to_ep_v2() API
+* Added new uct_mem_reg_v2() API
+* Added new uct_md_query_v2() API
+#### RDMA CORE (IB, ROCE, etc.)
+* Added ECE(enhanced connection establishment) support for RC and DC transports
+* Added support for hardware DCS in DC transport
+* Added UD iface and endpoint resource information to VFS
+* Added CQ creation via DEVX API
+* Removed support for accelerated IB transports over experimental verbs
```

Comment:
legacy support ?

Reply:
can you please elaborate?

Reply:
"Removed support for accelerated IB transports over legacy experimental verbs"